pages/category.html

```diff
@@ -0,0 +1,114 @@
+          <div>
+            <a href="">
+              <div>
+                <!--TODO insert shopify bopis image-->
+                <img src="../img/HC Logo.svg" class="logo">
+              </div>
+              <div>
+                <h3>Shopify Bopis</h3>
+                <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+              </div>
+            </a>
+          </div>
+        </div>
+
+        <div class="cards">
+          <div>
+            <a href="">
+              <div>
+                <!--TODO insert shopify preorder image-->
+                <img src="../img/HC Logo.svg" class="logo">
+              </div>
+              <div>
+                <h3>Shopify Preorder</h3>
+                <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+              </div>
+            </a>
+          </div>
+        </div>
+
+        <div class="cards">
+          <div>
+            <a href="">
+              <div>
+                <!--TODO insert shopify hc oms image-->
+                <img src="../img/HC Logo.svg" class="logo">
+              </div>
+              <div>
+                <h3>Shopify Hc Oms</h3>
+                <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+              </div>
+            </a>
+          </div>
+        </div>
+
+        <div class="cards">
+          <div>
+            <a href="">
+              <div>
+                <!--TODO insert shopify store mode image-->
+                <img src="../img/HC Logo.svg" class="logo">
+              </div>
+              <div>
+                <h3>Shopify Store Mode</h3>
+                <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+              </div>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
+
+<style>
+a {
+  text-decoration: none;
+}
+
+.logo {
+  width: auto;
+  height: 58px;
+}
+
+.banner-content,
+.card-header {
+  text-align: center;
+}
+
+.cards-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(275px, 1fr));
+  gap: 10px;
+  box-sizing: border-box;
+}
+
+.cards {
```

Comment:
manage card styling globally, we may want to use this later too. on this page app cards should have two classes, app and card. card will be styled globally and we will use app class to manage style here semantically.

Reply:
For card stying reference the cards at the bottom of the current home page of Hotwax.co

Reply:
Done